src/Management/test/Tracing.Test/TracingServiceCollectionExtensionsTest.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Steeltoe.Common;
+using Steeltoe.Logging;
+
+namespace Steeltoe.Management.Tracing.Test;
+
+public sealed class TracingServiceCollectionExtensionsTest
+{
+    [Fact]
+    public async Task AddTracingLogProcessor_RegistersTracingLogProcessor()
+    {
+        IServiceCollection services = new ServiceCollection();
+        services.AddSingleton<IConfiguration>(new ConfigurationManager());
```

Comment:
```suggestion
        services.AddSingleton<IConfiguration>(new ConfigurationBuilder().Build());
```
I hadn't seen usage of `ConfigurationManager` in similar tests before. Perhaps it doesn't matter much, but I prefer to use the existing syntax for consistency, and because the implementation is simpler, ie no auto-reloads etc.